_python/pgbench_python.py

```diff
@@ -247,7 +250,8 @@ def sync_worker(executor, eargs, start, duration, timeout):
             max_latency = req_time
         if req_time < min_latency:
             min_latency = req_time
-        latency_stats[req_time] += 1
+        if req_time <= 200000:
```

Comment:
Silently truncating latencies like that does not seem right.  Why is this change necessary?